src/converters/surface_converter.py

```
@@ -1,30 +1,46 @@
-            except Exception as e:
-                self.state['failed'] += 1
-                self.logger.error(f"Error Validate BuildingSurface Data: {e}", exc_info=True)
-                continue
-
-    
-    def _add_to_idf(self, data:Any) -> None:
+        surface_data = data.get("BuildingSurface:Detailed", [])
+        zone_to_surfaces = defaultdict(list)
+        for surface in surface_data:
+            zone_to_surfaces[surface["Zone Name"]].append(surface)
+        val_data = self.validate(zone_to_surfaces)
```

Ruff (0.14.1)

Local variable `val_data` is assigned to but never used

Remove assignment to unused variable `val_data`

(F841)